addons/addon-base-raas/packages/base-raas-workflows/lib/workflows/create-network-infra.yml

```diff
@@ -0,0 +1,17 @@
+id: wf-create-network-infra
+v: 1
+workflowTemplateId: wt-empty
+workflowTemplateVer: 1
+title: Create New Network Infra for Storage Gateway
```

Comment:
Can we add storage gateway in the file title as well?

Reply:
Didn't add it because this yaml wasn't doing that. Storage Gateway doesn't support CloudFormation currently.